src/ui/pages/Credentials/Credentials.tsx

```diff
@@ -121,7 +121,7 @@ const Credentials = () => {
       const creds = await Agent.agent.credentials.getCredentials(true);
       dispatch(setCredsArchivedCache(creds));
     } catch (e) {
-      // @TODO - duke: handle error
```

Comment:
Same here, if an error occurs when fetching archived credentials we should just log it, there is no toast message.